site/_posts/2023-09-14-manageiq-petrosian-ga-announcement.md

```diff
@@ -0,0 +1,89 @@
+    - Fix Node network metrics collection on s390x
+    - Improvements for data collection when provider connection is lost
+  - Kyndryl AutoSDE
+    - Basic and Advanced modes for storage manager volume creation
+    - Cloud Volumes Snapshot management
+    - Clone support for cloud volumes
+    - Update support for storage services
+    - Update and Delete support for host initiator groups
+    - Collection of storage service resource attachments and capabilities for physical storages / storage resources
+  - oVirt / Red Hat Virtualization
+    - Creation support for cloud network/subnets/routers though OVN/NetworkManager
+  - VMware vSphere
+    - Support for vSphere 8.0
+    - Added a customizable disk size field to VM post-provision
+- Core
+  - Added new "Workflows" engine prototype. This new feature uses Amazon States Language to define workflows that can be used to automate complex tasks, and is designed as a future replacement for Automate. Admins can create custom docker images that will execute the "tasks" within the workflow. This feature is currently in prototype and is not recommended for production use at this time, but will be released in the Quinteros release.
+  - Enable notifier role by default
+  - [Security] Read secrets from an ENV file if mounted in containerized deployments
+  - Performance improvements for metrics capture and rollups, particularly for very large container environments
+  - Performance improvements to evaluation of MiqExpressions
+  - Purging of audit_events
+- UI
+  - New Storage Services page
+  - Display Request Logs in Services -> Requests
+  - Added relative date as a formatting option for date columns in Reports
+  - Improved provider status notification when status has not been updated for over 2 days
+  - Updated Timeline Chart and Table Component
+  - Performance improvements for Timeline presentation when there are many events
+  - [Service UI] Add Native Console Support
+  - More page conversions from Angular to React and HAML to React
+- API
+ - TODO
+- Build
+  - Moved a lot of the build infrastructure to using GitHub Actions
+  - [Containerized] Add a new hotfix image build script allowing users to build their own new images with hotfixed RPMs
+  - [RPMs] Add tools for building hotfix RPMs
+- Developer
+  - Upgrade to Nodejs 18
+  - Upgrade to yarn 3.5.0
+  - Upgrade to go 1.20.8
+  - Upgrade to operator-sdk 1.22.1
+  - Foundational changes for a future upgrade to Rails 7
+
+**IMPORTANT**: The following are potentially breaking changes and deprecations
+
+- \*\***BREAKING**\*\*
+  - Removal of OCP v3 Support
+  - Removal of SCVMM provider
+  - Removal of Monitor menu and the underlying OpenShift alerts (These only worked with OCP v3, so with the removal of OCP v3, there are no alerts to monitor)
+  - API removal of /api/iso_datastores (these are now modeled as storages)
+- Deprecations carried over from the Oparin release
+  - VMware 5.x and below support
```

Comment:
@agrare I wasn't sure about this one, or if we can just mark it as "support removed"

Reply:
Discussed offline, and we actively block <6.0, so move this to "removed"